plugins/auth-backend/src/providers/okta/provider.ts

```diff
@@ -230,6 +233,7 @@ export const okta = createAuthProviderIntegration({
         const callbackUrl =
           customCallbackUrl ||
           `${globalConfig.baseUrl}/${providerId}/handler/frame`;
+        const scope = envConfig.getOptionalString('scope');
```

Comment:
every config parameter used should also be represented [in the config schema](https://github.com/backstage/backstage/blob/4e15d5e4bfe00e672df06e2e3d82a78fc30e5218/plugins/auth-backend/config.d.ts#L138)! Remember to make it optional.